app/js/components/ArtifactGenerator.jsx

```diff
@@ -97,11 +98,10 @@ const ArtifactGenerator = ({
                             />
                         </td>
                         <td style={style}>
-                            <input
+                            <ReactSelect
                                 name="source_format"
```

Comment:
Keeping the `name` as is lets the `Form` get the value just as it did when this was a text input.